fuel-core/src/database/message.rs

```diff
@@ -78,27 +78,14 @@ impl Database {
         })
     }
 
-    pub fn all_owners_and_message_ids(
+    pub fn all_messages(
         &self,
-        start: Option<OwnerAndMessageId>,
+        start: Option<MessageId>,
```

Comment:
good one, I forgot that we could just query by ID instead of combing the owner and the ID